i6_models/parts/factored_hybrid/triphone.py

```diff
@@ -0,0 +1,69 @@
+__all__ = [
+    "FactoredTriphoneBlockV1Config",
+    "FactoredTriphoneBlockV1",
+]
+
+from dataclasses import dataclass
+from typing import Tuple
+
+import torch
+from torch import nn, Tensor
+
+from .diphone import FactoredDiphoneBlockV1, FactoredDiphoneBlockV2Config
+from .util import get_mlp
+
+
+@dataclass
+class FactoredTriphoneBlockV1Config(FactoredDiphoneBlockV2Config):
+    """
+    Attributes:
+        Same as the FactoredDiphoneBlockV2Config.
+    """
```

Comment:
would `from .diphone import FactoredDiphoneBlockV2Config as FactoredTriphoneBlockV1Config` also work?

Reply:
It would, but I kind of would like this to be a real type instead of only an alias, in case we ever need to add more properties? Maybe it shouldn't inherit from the other class at all then -- but that would also reduce DRY-ness. 🤔